repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/AccessProfileEntity.java

```diff
@@ -38,7 +38,7 @@
 public class AccessProfileEntity implements Serializable {
     @Id
     @Column(name = "id")
-    @GeneratedValue(strategy = IDENTITY)
+    @GeneratedValue(strategy = SEQUENCE, generator = "role_id_seq")
```

Comment:
Just wonder if you have considered `pooled-lo` optimiser already? 
`<property name="hibernate.id.optimizer.pooled.preferred" value="pooled-lo" />`
https://vladmihalcea.com/hibernate-hidden-gem-the-pooled-lo-optimizer/

Reply:
I haven't seen this Sateesh, will read it and try, then compare the performance. I will let you know